src/app/store-stats/StatTooltip.tsx

```diff
@@ -13,8 +16,23 @@ interface Stat {
   breakdown?: DimCharacterStatChange[];
 }
 
-function StatTooltip({ stat }: { stat: Stat }) {
+/**
+ * A rich tooltip for character-level stats like Mobility, Intellect, etc.
+ */
+export default function StatTooltip({
+  stat,
+  equippedHashes,
+}: {
+  stat: Stat;
+  /**
+   * Hashes of equipped/selected items and subclass plugs for this character or loadout. Can be limited to
+   * exotic armor + subclass plugs - make sure to include default-selected subclass plugs.
+   */
+  equippedHashes: Set<number>;
+}) {
   const tier = statTier(stat.value);
+  const descriptionsToDisplay = useSelector(settingSelector('descriptionsToDisplay'));
+  const useClarityInfo = descriptionsToDisplay !== 'bungie';
```

Comment:
Should we also check the Clarity descriptions feature flag here?

```suggestion
  const useClarityInfo = $featureFlags.clarityDescriptions && descriptionsToDisplay !== 'bungie';
```

Reply:
Also, should we hide the Bungie-provided stat description if the user has set `Community Perk Insights` to `Community Descriptions`?

Reply:
Probably not for now.
I do plan to add a "community description" to the stats themselves at a later date and maybe you could do it at that point?